skyportal/utils/offset.py

```diff
@@ -32,6 +36,90 @@
 log = make_log('finder-chart')
 
 
+class GaiaQuery:
+
+    alt_tap = 'https://gaia.aip.de/tap'
```

Comment:
Maybe make this configurable?

Reply:
Let's refactor this once we also have a pathway to querying Kowalski (ie in another PR).